src/nix-eval-jobs.cc

```diff
@@ -59,6 +60,15 @@ struct MyArgs : MixEvalArgs, MixCommonArgs
             }},
         });
 
+        addFlag({
+            .longName = "depth",
```

Comment:
I guess this is nice to preserve backwards compatibility but it should be also documented in the README.